linux_os/guide/system/accounts/accounts-session/accounts_tmout/oval/shared.xml

```diff
@@ -40,6 +44,46 @@
   {{{ test_tmout(  test_stem="etc_profiled_tmout", files="/etc/profile.d/*.sh") }}}
   {{{ object_tmout(test_stem="etc_profiled_tmout", path="/etc/profile.d", filename="^.*\.sh$") }}}
 
+  {{% if product in ["ol7", "rhel7"] %}}
+  {{{ test_tmout(  test_stem="etc_bashrc_tmout", files="/etc/bashrc") }}}
+  {{{ object_tmout(test_stem="etc_bashrc_tmout", filepath="/etc/bashrc") }}}
+  {{% endif %}}
+
+  {{% if product in ["ol7", "rhel7"] %}}
+  <ind:textfilecontent54_object id="object_accounts_tmout_all_tmout_instances" version="1">
```

Comment:
Readability of the OVAL has been reduced, mainly because what used to be "all" became just partial, and a new "all" has been introduced. Would it be possible not to introduce `object_accounts_tmout_partial_tmout_instances` and only to have an "if bashrc is a thing, then object_accounts_tmout_all_tmout_instances should also consider object_etc_bashrc_tmout"?

Reply:
Unfortunately some trickery needs to be done as `<set>` can only be composed of two child `object_references`. 
I cannot reference 3 objects, so I need to compose them.
Not very smart from the OVAL part...

Reply:
I see, I wasn't aware of the 2-limit. In that case, I would suggest to change the approach a bit, when you define both partial + all in one condition branch, and all in the second (old) one. That way, the new if would bring complete entities, and not a new entity and just a new "header" of the old one. Although there would be text that is identical in both of them, I think that it is a worthy tradeoff.

Reply:
Changes done in [8c7107c](https://github.com/ComplianceAsCode/content/pull/9915/commits/8c7107cd17e82181f72f631d82c9b4657b848951)